Library/Formula/rcssserver.rb

```diff
@@ -67,7 +67,31 @@ def install
 
     resources.each do |r|
       r.stage do
-        system "./bootstrap" if build.head?
+        Patch.create(:p1, <<EOD).apply
```

Comment:
We don't currently support patching resources. We may need to wait for an upstream release.